bookings/models.py

```diff
@@ -28,18 +29,34 @@ def create_reservation(
         api = TicketingSystemAPI(ticketing_system, maas_operator)
         response_data = api.reserve(ticket_data)
 
-        return Booking.objects.create(
+        booking = Booking.objects.create(
             source_id=response_data["id"],
             maas_operator=maas_operator,
             ticketing_system=ticketing_system,
             route_name=route_name,
             ticket_count=ticket_count,
             transaction_id=transaction_id,
+            locale=ticket_data["locale"],
+            route_capacity_sales=Route.CapacitySales(ticket_data['route'].capacity_sales).label,
+            agency_name=ticket_data["agency"]["name"]
         )
+        for ticket in ticket_data["tickets"]:
+            Ticket.objects.create(
+                booking=booking,
+                customer_type_name=ticket["fare_rider_category"].name,
+                price=ticket["fare"].price,
+                currency_type=ticket["fare"].currency_type,  # TODO: Remove if not required
+                ticket_type_name=ticket["fare"].name
+            )
+
+        return booking
 
 
+#  TODO: We've created_at auto_now field, but required is ticket_data['created_at'],
```

Comment:
Yes that's a good idea. As we wan't to grab the `created_at` from the ticket data coming back from the ticket system it's a good idea to store that to `booking_created_at` field.